ui/pages/home/beta/beta-home-footer.component.js

```diff
@@ -13,15 +13,16 @@ const BetaHomeFooter = () => {
   const t = useI18nContext();
   const trackEvent = useContext(MetaMetricsContext);
 
-  const SUPPORT_LINK =
-    'https://docs.google.com/forms/d/e/1FAIpQLSfpkpeVMvfFw_1CWzrs5M3TEqE9VPMSspLPJgk1u4ZyUkQRbg/viewform?usp=sharing';
+  // TODO: Bring back once internal beta is over
+  // const SUPPORT_LINK =
+  //   'https://docs.google.com/forms/d/e/1FAIpQLSfpkpeVMvfFw_1CWzrs5M3TEqE9VPMSspLPJgk1u4ZyUkQRbg/viewform?usp=sharing';
 
   return (
     <>
       <a
         target="_blank"
         rel="noopener noreferrer"
-        href={SUPPORT_LINK}
+        href={SUPPORT_REQUEST_LINK}
```

Comment:
I would use `SUPPORT_LINK` here and import it from `../../../helpers/constants/common` (since it seems that both links might serve different purposes)